.yarn/patches/modular-scripts-npm-3.1.0-5a349ac39e.patch

```diff
@@ -77,20 +61,6 @@ index 4f6555e3988e6a229a64e2c429107e0f62526d39..d8ef7201d651d92262f1cc422f1136a4
      'node_modules', 'bower_components', 'jspm_packages', 'tmp', // Casting so that configObject.exclude is set to the correct typing
      // Since configObject is a index type all values are "any" implicitly.
      ...(configObject.exclude || [])])
-diff --git a/react-scripts/config/paths.js b/react-scripts/config/paths.js
-index 5587b7cda8a2841b4fd7c2950127cc3a102f3f19..905456cb8e02b32b4f105da274a3d4d72ddca440 100644
---- a/react-scripts/config/paths.js
-+++ b/react-scripts/config/paths.js
-@@ -88,6 +88,9 @@ module.exports = {
-   appSrc: resolveApp('src'),
-   modularSrc: [
-     resolveModular('packages'),
-+	resolveModular('playground'),
-+	resolveModular('tooling'),
-+	resolveModular('docs'),
```

Comment:
No `resolveModular('design-tools')`?

Reply:
Since it's under the packages workspace it's not needed.